modular_zubbers/modules/ballstics/wt_autorifle.dm

```diff
@@ -0,0 +1,209 @@
+	max_ammo = 20
+
+/obj/item/ammo_box/magazine/wt550m9/rubber
+	name = "wt550 magazine (Rubber 4.6x30mm)"
+	icon_state = "46x30mmt-20"
+	base_icon_state = "46x30mmt"
+	ammo_type = /obj/item/ammo_casing/c46x30mm/rubber
+	caliber = CALIBER_46X30MM
+	max_ammo = 20
+
+
+//ORDERING
+/datum/supply_pack/security/armory/wt551
+	name = "WT-551 Autorifle Crate"
+	desc = "Contains a pair of WT-551 Autorifles pre-loaded with less-than-lethal rubber-tipped rounds. Additional ammo sold seperately. Backwards-compatible with WT-550 magazines. NanoTrasen reminds you that the other weapon is for a friend, and not for going guns akimbo."
+	cost = CARGO_CRATE_VALUE * 8
+	contains = list(/obj/item/gun/ballistic/automatic/wt550/security/rubber = 2)
+	crate_name = "wt-550 autorifle crate"
+
+/datum/supply_pack/security/armory/wt550_ammo_rubber
+	name = "WT-550/WT-551 Autorifle Ammo Crate (Rubber-Tipped)"
+	desc = "Contains 3 magazines with less than lethal rubber-tipped rounds for the WT-551."
+	cost = CARGO_CRATE_VALUE * 4
+	contains = list(/obj/item/ammo_box/magazine/wt550m9/rubber = 3)
+	crate_name = "wt-550 magazine crate (rubber-tipped)"
+
+/datum/supply_pack/security/armory/wt550_ammo_flat
+	name = "WT-550/WT-551 Autorifle Ammo Crate (Flat-Tipped)"
+	desc = "Contains 3 magazines with lethal flat-tipped rounds for the WT-551."
+	cost = CARGO_CRATE_VALUE * 5
+	contains = list(/obj/item/ammo_box/magazine/wt550m9/flathead = 3)
+	crate_name = "wt-550 magazine crate (flat-tipped)"
+
+/datum/supply_pack/security/armory/wt550_ammo_regular
+	name = "WT-550/WT-551 Autorifle Ammo Crate (Regular)"
+	desc = "Contains 3 magazines with lethal flat-tipped rounds for the WT-551."
+	cost = CARGO_CRATE_VALUE * 7
+	contains = list(/obj/item/ammo_box/magazine/wt550m9 = 3)
+	crate_name = "wt-550 magazine crate (regular)"
+
+//RESEARCHING: TECHWEBS
+
+/datum/techweb_node/wt550_ammo
+	id = "wt550_ammo"
+	display_name = "Basic WT-550/WT-551 Ammunition"
+	description = "They won't know what hit em."
+	prereq_ids = list("weaponry")
+	design_ids = list(
+		"wt550_ammo_rubber",
+		"wt550_ammo_flathead"
+	)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+
+/datum/techweb_node/wt550_ammo_advanced
+	id = "wt550_ammo_advanced"
+	display_name = "Advanced WT-550/WT-551 Ammunition"
+	description = "They'll really won't know what hit em."
+	prereq_ids = list("adv_weaponry","wt550_ammo")
+	design_ids = list(
+		"wt550_ammo_normal",
+		"wt550_ammo_ap"
+	)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
+
+
+/datum/techweb_node/wt550_ammo_exotic
+	id = "wt550_ammo_exotic"
+	display_name = "Illegal WT-550/WT-551 Ammunition"
+	description = "They'll REALLY won't know what hit em. Or they will, because they'll be on fire and likely reporting that warcrime to the government in your sector."
+	prereq_ids = list("exotic_ammo","syndicate_basic","wt550_ammo_advanced")
+	design_ids = list(
+		"wt550_ammo_incendiary"
+	)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 7500)
```

Comment:
There's no reason to make three separate tech nodes. Just throw them into ``modular_zubbers\code\modules\research\techweb\all_nodes.dm`` under pre-existing tech webs with the ``/datum/techweb_node/New()`` proc.